### ISSUE TITLE
ERB does not evaluate no matter what I try

### ISSUE THREAD

**Issue by martinvelos** [tmuxinator/tmuxinator#731]
**Describe the bug**
After not being able to insert the @args[0], I tried whatever to track the problem, this is the minimal project:
```
# ~/.tmuxinator/playground.yml
name: playground
root: ~/
windows:
  - content:
      layout: vertical
      panes:
        #- echo $varr; echo <%= settings["par"] %> 
        - echo $varr; echo <%= args[0] %>
        - bash --version
```
and when I run it, with a parameter or not, I get:  
```
$ tmuxinator start playground paraaam
Failed to parse config file. Please check your formatting.
```

**Expected behavior**
A more guiding message with what to fix or where the problem was detected.
Or just ERB to work as in the https://github.com/tmuxinator/tmuxinator#erb .

**Environment**
 - tmuxinator Version: tmuxinator 0.6.11 # Hmm, Fedora has some archaic version?
- tmux Version: tmux 2.9a
 - Ruby Version: ruby 2.6.3p62 (2019-04-16 revision 67580) [x86_64-linux]
 - OS: Fedora release 30 (Thirty)

**Comment by martinvelos** [comment]
OK, I just fixed it by installing from gem. I think we should push on Fedora maintainers to either remove the ancient tmuxinator or better update it.

**Comment by Dridi** [comment]
TL;DR: I may maintain `tmuxinator` for Fedora (I'm strongly considering it) and might forward bug reports here in the future, and would hope to see an update to use `xdg` 4.x soon.

Apologies for hijacking this issue but I felt it was a great place to introduce myself as required by Fedora packaging guidelines, considering the previous comment.

`tmuxinator` disappeared from Fedora as per version 32, soon to be released. There is no longer a maintainer and as a user I'm willing to give a it a try. I usually don't maintain Ruby packages since I'm not a Ruby developer and won't be able to do more than forward bug reports here if I get any. I was nevertheless able to package `tmuxinator` 1.1.4 on my Fedora 31 laptop and will extensively use it parallel to the packaging process to self QA.

I had to also package Ruby gems `thor` 1.0.1 and `xdg` 2.2.5 and so far that's fine because those being missing dependencies implies that once I submit `tmuxinator` it will be the sole consumer. However, as per Fedora principles I should nag^Wencourage you to update your dependencies to the latest version, 4.1.0 as of today.

Should I open an issue for that? Would it be welcome?

**Comment by ethagnawl** [comment]
> Should I open an issue for that? Would it be welcome?

Sure. That would be great.

I just took a quick look through the xdg gem's changelog and the one issue that sticks out is that it requires Ruby `~> 2.7`. Tmuxinator has historically tracked the currently maintained versions of Ruby (we're currently one and soon to be two minor versions behind -- I'll address this separately). So, we can't jump ahead to xdg 4.1.1, but we can bump up to 3.0.1 and then move incrementally as Ruby versions are EOL'd. (I'm happy to continue hashing this out in your new issue.)
